### PR TITLE
Add CrowdStrike filebeat module

### LINKED ISSUE #183: Crowdstrike Filebeat Module
Checklist for imported CrowdStrike `filebeat` module:

- [x] initial import
- [x] additional ECS fields
- [x] inlining pipeline
- [x] clean up manifest
- [x] documentation
- [ ] testing

### PR DESCRIPTION
**Pull request by andrewstucki**
## What does this PR do?

~~So, one of the things in the handlebars yaml file to bring it to parity with what's currently in filebeat is blocked by https://github.com/elastic/kibana/pull/72698~~

~~I'm going to try and actually test this locally as well, but I'm putting this up here early in case anyone wants to give early feedback.~~

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [x] I have verified that all datasets collect metrics or logs.

## Related issues

- Closes #183

## Screenshots

Screen Shot 2020-07-21 at 4 54 04 PM
Screen Shot 2020-07-21 at 4 54 13 PM
Screen Shot 2020-07-21 at 4 54 20 PM
Screen Shot 2020-08-04 at 11 23 00 AM
Screen Shot 2020-08-04 at 11 23 17 AM
Screen Shot 2020-08-04 at 12 12 45 PM

### CODE REVIEW

File: packages/crowdstrike/dataset/falcon/agent/stream/log.yml.hbs [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+multiline.pattern: '^{'
+multiline.negate: true
+multiline.match: after
+multiline.max_lines: 5000
+multiline.timeout: 10
+tags:
+{{#each tags as |tag i|}}
+ - {{tag}}
+{{/each}}
+{{#contains tags "forwarded"}}
```
Comment (andrewstucki) [inline packages/crowdstrike/dataset/falcon/agent/stream/log.yml.hbs:16]: This syntax comes from elastic/kibana#72698 -- not entirely sure if it's needed here. When agent manages filebeat do we know if it comes with global processors enabled? Currently in beats this tag allows for skipping host name enrichment in the global `add_host_metadata` processor.

File: packages/crowdstrike/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- src: /img/logo-integrations-crowdstrike.svg
+  title: CrowdStrike
+  size: 216x216
+  type: image/svg+xml
+screenshots:
+- src: /img/siem-alerts-cs.jpg
+  title: CrowdStrike SIEM Alerts
+  size: 3360x1776
+  type: image/jpg
+- src: /img/siem-events-cs.jpg
```
Comment (andrewstucki) [inline packages/crowdstrike/manifest.yml:22]: Should this appear in the Kibana UI? It looks like it's only showing the first screenshot right now?

Comment (andrewkroh) [reply packages/crowdstrike/manifest.yml]: I noticed the same thing with Suricata. I kind of expected to be able to thumb through the screenshots.

File: packages/crowdstrike/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            title: Tags
+            multi: true
+            required: true
+            show_user: true
+            default:
+              - forwarded
+        title: "Collect CrowdStrike Falcon logs (input: logfile)"
+        description: "Collecting logs from CrowdStrike Falcon (input: logfile)"
+owner:
+  github: elastic/integrations
```
Comment (andrewkroh) [inline packages/crowdstrike/manifest.yml:44]: I created an @ elastic/security-ingest for us to use in security modules.

Comment (andrewstucki) [reply packages/crowdstrike/manifest.yml]: cool, followed suit

**Pull request merged**